easybuild/easyblocks/f/fsl.py

```diff
@@ -63,22 +66,47 @@ def configure_step(self):
         fslmachtype = out.strip()
         self.log.debug("FSL machine type: %s" % fslmachtype)
 
-        # prepare config
-        # either using matching config, or copy closest match
+        # Patch files for ver. < 5.0.10 patch multiple config directories
+        if LooseVersion(self.version) >= LooseVersion('5.0.10'):
+            # Check if a specific machine type directory is patched
+            systype_regex = re.compile("^diff.*config\/(.*(apple|gnu|i686|linux|spark)(?:(?!\/).)*)", re.M)
+
+            patched_cfgs = []
+
+            for patch in self.patches:
+                patchfile = read_file(patch['path'])
+                res = systype_regex.findall(patchfile)
+                patched_cfgs.extend([i[0] for i in res])
+
+            # Check that at least one config has been found
+            if patched_cfgs:
+                # Check that a single config has been patched
+                if len(nub(patched_cfgs)) == 1:
+                    best_cfg = patched_cfgs[0]
+                    self.log.debug("Found patched config dir: %s", best_cfg)
+                else:
+                    raise EasyBuildError("Patch files are editing multiple config dirs: %s", patched_cfgs)
+            else:
+                self.log.debug("No config dir found in patch files")
+
+        # If no patched config is found, pick best guess
         cfgdir = os.path.join(self.fsldir, "config")
         try:
-            cfgs = os.listdir(cfgdir)
-            best_cfg = difflib.get_close_matches(fslmachtype, cfgs)[0]
-
-            self.log.debug("Best matching config dir for %s is %s" % (fslmachtype, best_cfg))
-
-            if fslmachtype != best_cfg:
-                srcdir = os.path.join(cfgdir, best_cfg)
-                tgtdir = os.path.join(cfgdir, fslmachtype)
-                shutil.copytree(srcdir, tgtdir)
-                self.log.debug("Copied %s to %s" % (srcdir, tgtdir))
-        except OSError, err:
-            raise EasyBuildError("Failed to copy closest matching config dir: %s", err)
+            if not best_cfg:
+                cfgs = os.listdir(cfgdir)
+                best_cfg = difflib.get_close_matches(fslmachtype, cfgs)[0]
+                self.log.debug("Best matching config dir for %s is %s" % (fslmachtype, best_cfg))
+        except OSError as err:
+            raise EasyBuildError("Unable to access configuration directory: %s", cfgdir, err)
```

Comment:
@vanzod You have a `%s` missing here, this is going to result in a crash rather than an error message, should be:

```python
raise EasyBuildError("Unable to access configuration directory %s: %s", cfgdir, err)